lib/src/base-styles/_nx-card.scss

```diff
@@ -0,0 +1,173 @@
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-card-container {
+  display: flex;
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
+
+.nx-card {
+  background-color: #fff;
+  border-radius: $nx-border-radius;
+  box-sizing: border-box;
+  padding: $nx-spacing-l $nx-spacing-xxl;
+}
+
+.nx-card__call-out {
+  .nx-icon {
+    font-size: 48px;
+  }
+
+  .nx-icon--large {
+    font-size: 64px;
+  }
+}
+
+.nx-call__call-out-text {
+  @include bold();
+
+  font-size: 32px;
+}
+
+// Vertically oriented cards appear in a row container
+.nx-card-container--row {
+  @include container-horizontal;
+
+  .nx-card {
+    display: flex;
+    flex-direction: column;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    margin: 0 $nx-spacing-md 0 0;
+    text-align: center;
+    min-width: 230px;
+    max-width: 330px;
+  }
+
+  .nx-card-header {
+    @include container-vertical;
+    margin-bottom: $nx-spacing-md;
+  }
+
+  .nx-card__call-out {
+    margin-bottom: $nx-spacing-xs;
+  }
+
+  .nx-card__text {
+    margin-bottom: $nx-spacing-l;
+  }
+
+  .nx-card-footer {
+    border-top: 1px solid #d1d6f0;
+    padding-top: $nx-spacing-l;
+    width: 100%;
+  }
+}
+
+// Horizontally orientated cards appear in a column
+.nx-card-container--column {
+  @include container-vertical;
+
+  flex-direction: column;
+
+  .nx-card {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: auto 1fr auto;
+    grid-template-columns: auto 1fr auto;
+    -ms-grid-rows: auto auto;
+    grid-template-areas:
+      "card-callout card-title card-footer"
+      "card-callout card-text card-footer";
+
+    margin: $nx-spacing-md 0;
+    text-align: left;
+    width: 100%;
+  }
+
+  .nx-card-header {
+    -ms-grid-row: 1;
+    -ms-grid-column: 2;
+    grid-area: card-title;
+
+    @include container-vertical;
+    border-right: 1px solid #d1d6f0;
+    border-left: 1px solid #d1d6f0;
+    margin: 0;
+    padding: 0 $nx-spacing-xxl $nx-spacing-md $nx-spacing-xxl;
+  }
+
+  .nx-card__text {
+    -ms-grid-row: 2;
+    -ms-grid-column: 2;
+    grid-area: card-text;
+
+    border-right: 1px solid #d1d6f0;
+    border-left: 1px solid #d1d6f0;
+    margin: 0;
+    padding: 0 $nx-spacing-xxl;
+  }
+
+  .nx-card__call-out {
+    -ms-grid-row-align: center;
+    align-self: center;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 1;
+    grid-area: card-callout;
+    -ms-grid-column-align: center;
+    justify-self: center;
+
+    margin: 0;
+    padding: 0 $nx-spacing-xxl 0 0;
+  }
+
+  .nx-card-footer {
+    -ms-grid-row-align: center;
+    align-self: center;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 3;
+    grid-area: card-footer;
+    -ms-grid-column-align: start;
+    justify-self: start;
+
+    border: none;
+    margin: 0;
+    padding: 0 0 0 $nx-spacing-xxl;
+    // necessary to make the link wrap
+    max-width: 150px;
```

Comment:
There's a mockup with two icon buttons in the footer, which brings it to 136px.  I'm wondering what should happen when there are more than two buttons.  I'm assuming you came up with 150px here arbitrarily; when does @dturner32 think buttons or link text should wrap?

Reply:
> There's a mockup with two icon buttons in the footer, which brings it to 136px. I'm wondering what should happen when there are more than two buttons. I'm assuming you came up with 150px here arbitrarily; when does @dturner32 think buttons or link text should wrap?

Ah, I missed the examples with the icon-only buttons. The reason for this value is because without it the text never wrapped and the column would stretch far further than we wanted

Reply:
> The reason for this value is because without it the text never wrapped and the column would stretch far further than we wanted

I know, but my question is whether 150 is in fact the right width at which to wrap.

Reply:
So I just tested a three button layout in the design as a quick mock and the total width of those buttons plus the padding came out to 144px so Glen's suggestion for 150px isn't too far off. I'm ok with the 150px max